import-service/src/handlers/catalogBatchProcesss.js

```diff
@@ -0,0 +1,18 @@
+const AWS = require('aws-sdk');
+
+exports.catalogBatchProcesss = (event, context) => {
```

Comment:
You missed logic of storing sent data:
The lambda function should iterate over all SQS messages and create corresponding products in the products table.